docs/_data/topnav.yml

```diff
@@ -3,6 +3,8 @@
 topnav:
 - title: Topnav
   items:
+    - title: Releases
+      url: /releases.html
     - title: News
```

Comment:
this will make the topnav very busy.. can you remove News for now.. ? we will figure it out later